scopes/design/ui/external-link/external-link.docs.mdx

```diff
@@ -0,0 +1,21 @@
+---
+labels: ['react', 'typescript', 'link', 'external']
+description: 'A link component to display an external link with no text decoration'
+---
+
+import { ThemeCompositions } from '@teambit/documenter.theme.theme-compositions';
+import { ExternalLink } from './external-link';
+
+### Overview
```

Comment:
Not need to add "Overview" if there is not a lot of text and titles to write in the docs file. (Like in aspect docs for example)